pkg/ddl/placement_policy_test.go

```diff
@@ -2401,3 +2401,15 @@ func TestRecoverTableWithPlacementPolicy(t *testing.T) {
 		" PARTITION `p2` VALUES LESS THAN (10000))"))
 	checkExistTableBundlesInPD(t, dom, "test", "tp3")
 }
+
+func TestPartitionByWithLabels(t *testing.T) {
+	store, _ := testkit.CreateMockStoreAndDomain(t)
+	tk := testkit.NewTestKit(t, store)
+	tk.MustExec("use test")
+
+	tk.MustExec("create placement policy pp1 primary_region='r1' regions='r1,r2'")
+
+	tk.MustExec(`CREATE TABLE t1 (id INT)`)
+	tk.MustExec(`ALTER TABLE t1 placement policy pp1`)
+	tk.MustExec(`ALTER TABLE t1 PARTITION BY HASH (id) PARTITIONS 3`)
```

Comment:
Not sure, but it looks like this sql will not fail in a unistore environment

Reply:
If removing the change in `partition.go` it will fail in the check in `(* mockPlacementManager) PutRuleBundles()`